### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by csadorf**
Fixes issue #4 .

### CODE REVIEW

**Comment by csadorf** [comment]
@ltalirz Is this roughly what you had in mind or anything more involved?

File: README.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ The AiiDAlab setup is based on [z2jh](https://zero-to-jupyterhub.readthedocs.io/
 
 This sets up the environment of your local machine for deployment.
 
+*Note: The instructions below assume that you already you have a kubernetes cluster available for the deployment of AiiDAlab.*
+*However, you can use [terraform](./terraform/README.md) to setup a cluster suiteable for an AiiDAlab (JupyterHub) deployment if needed.*
```
Comment (ltalirz) [inline README.md]:
```suggestion
*However, you can use [terraform](./terraform/README.md) to setup a cluster suitable for an AiiDAlab (JupyterHub) deployment if needed.*
```

**Review by ltalirz — COMMENTED** [review]
thanks, simon!

I think that may be enough

File: README.md (hunk 1 above)
Comment (csadorf) [reply README.md]: Thx for the catch!

**Pull request merged**